setup.py

```diff
@@ -11,7 +11,7 @@
 
 tests_requires = [
     'pytest',
-    'coverage==4.5.3',
+    'coverage==3.6.0',
```

Comment:
What's the reasoning behind this change?

Reply:
Yup, sorry...probably forgot to switch branches before pushing